qa/L0_backend_python/examples/test.sh

```diff
@@ -38,10 +38,25 @@ rm -fr *.log python_backend/
 
 # Install torch
 pip3 uninstall -y torch
+pip3 uninstall -y numpy
+# NOTE: Using this subtest as a test case that involves using a python model with
+# numpy 2.X without changing the environments used in all the other test cases.
 if [ "$TEST_JETSON" == "0" ] && [[ ${TEST_WINDOWS} == 0 ]]; then
-    pip3 install torch==2.0.0+cu117 -f https://download.pytorch.org/whl/torch_stable.html torchvision==0.15.0+cu117
+    if [ ${PYTHON_ENV_VERSION} == "8" ]; then
```

Comment:
Comment why you're splitting it for future readers. ex: python 3.8 doesn't support a version of torch compatible with numpy 2.x